steps/applivery-deploy-ios/0.1/step.yml

```diff
@@ -0,0 +1,119 @@
+title: Applivery.com iOS Deploy
+summary: Deploy your awesome iOS Application to Applivery.com
+description: |-
+  Deploy an iOS application to [Applivery](http://www.applivery.com),
+    add notes and notify testers.
+
+    Register a Applivery account at [http://www.applivery..com/](http://www.applivery.com)
+    and create an App to utilize this step.
+
+    You also need to get your *Account API Key* for you account and the *App Id* for the app.
+website: https://github.com/applivery/steps-applivery-ios-deploy
+source_code_url: https://github.com/applivery/steps-applivery-ios-deploy
+support_url: https://github.com/applivery/steps-applivery-ios-deploy/issues
+published_at: 2016-02-07T16:44:09.038396058+01:00
+source:
+  git: https://github.com/applivery/steps-applivery-ios-deploy.git
+  commit: aca3a18df31de473bfc1c24b49a3d9d05a275c8e
+host_os_tags:
+- osx-10.10
+project_type_tags:
+- ios
+type_tags:
+- deploy
+- Applivery
+dependencies:
```

Comment:
Actually, you can drop the `git` dependency all together - `git` is a dependency of the `bitrise CLI`.